src/cliAction/buildDeploy/handler_push.go

```diff
@@ -32,24 +32,29 @@ func (h *Handler) Push(ctx context.Context, config RunConfig) error {
 
 	buildConfigContent, err := func() ([]byte, error) {
 		for _, file := range files {
-			if file.ArchivePath == "zerops.yml" {
-				buildConfigContent, err := ioutil.ReadFile(file.SourcePath)
+			if file.ArchivePath == zeropsYamlFileName {
+				stat, err := os.Stat(file.SourcePath)
 				if err != nil {
 					return nil, err
 				}
 
-				if len(buildConfigContent) == 0 {
-					return nil, errors.New(i18n.BuildDeployBuildConfigEmpty)
+				if stat.Size() == 0 {
+					return nil, errors.New(i18n.BuildDeployZeropsYamlEmpty)
 				}
-				if len(buildConfigContent) > 10*1024*1024 {
-					return nil, errors.New(i18n.BuildDeployBuildConfigTooLarge)
+				if stat.Size() > 10*1024 {
```

Comment:
Is this enforced on the backend also somehow?

Reply:
Yep